app/Http/Controllers/DraftController.php

```diff
@@ -263,7 +262,7 @@ public function process(Request $request, Draft $draft)
                     $study = Study::create([
                         'name' => $folder->name,
                         'slug' => Str::slug($folder->name, '-'),
-                        'description' => $folder->name,
+                        'description' => 'This study contains NMR spectra obtained for the sample: '.$folder->name,
```

Comment:
Please don't populate the description. Keep it empty when its optional
'description' => ''